src/pages/member_resources.mdx

```diff
@@ -0,0 +1,41 @@
+import { Button } from "../components/button";
+
+# **Member Resources**
+
+### **LGBTQ Resources**
```

Comment:
Can you fix the hierarchy of headings? `#` should only have `##` below it (i.e, it's bad™ to go from `#`/`h1` to `###`/`h3`).